traffic_ops/traffic_ops_golang/server/servers_test.go

```diff
@@ -178,199 +178,6 @@ func TestGetServersByCachegroup(t *testing.T) {
-
-	cgs := []tc.CacheGroup{}
-	testCG1 := tc.CacheGroup{
-		ID:                          1,
-		Name:                        "Cachegroup",
-		ShortName:                   "cg1",
-		Latitude:                    38.7,
-		Longitude:                   90.7,
-		ParentCachegroupID:          2,
-		SecondaryParentCachegroupID: 2,
-		LocalizationMethods: []tc.LocalizationMethod{
-			tc.LocalizationMethodDeepCZ,
-			tc.LocalizationMethodCZ,
-			tc.LocalizationMethodGeo,
-		},
-		Type:        "EDGE_LOC",
-		TypeID:      6,
-		LastUpdated: tc.TimeNoMod{Time: time.Now()},
-		Fallbacks: []string{
-			"cachegroup2",
-			"cachegroup3",
-		},
-		FallbackToClosest: true,
-	}
-	cgs = append(cgs, testCG1)
-	testCG2 := tc.CacheGroup{
-		ID:                          2,
-		Name:                        "parentCacheGroup",
-		ShortName:                   "pg1",
-		Latitude:                    38.7,
-		Longitude:                   90.7,
-		ParentCachegroupID:          1,
-		SecondaryParentCachegroupID: 1,
-		LocalizationMethods: []tc.LocalizationMethod{
-			tc.LocalizationMethodDeepCZ,
-			tc.LocalizationMethodCZ,
-			tc.LocalizationMethodGeo,
-		},
-		Type:        "MID_LOC",
-		TypeID:      7,
-		LastUpdated: tc.TimeNoMod{Time: time.Now()},
-	}
-	cgs = append(cgs, testCG2)
-
-	ts := servers[1]
-	rows2 = rows2.AddRow(
-		ts.Cachegroup,
-		ts.CachegroupID,
-		ts.CDNID,
-		ts.CDNName,
-		ts.DomainName,
-		ts.GUID,
-		ts.HostName,
-		ts.HTTPSPort,
-		ts.ID,
-		ts.ILOIPAddress,
-		ts.ILOIPGateway,
-		ts.ILOIPNetmask,
-		ts.ILOPassword,
-		ts.ILOUsername,
-		ts.InterfaceMtu,
-		ts.InterfaceName,
-		ts.IP6Address,
-		ts.IP6IsService,
-		ts.IP6Gateway,
-		ts.IPAddress,
-		ts.IPIsService,
-		ts.IPNetmask,
-		ts.IPGateway,
-		ts.LastUpdated,
-		ts.MgmtIPAddress,
-		ts.MgmtIPGateway,
-		ts.MgmtIPNetmask,
-		ts.OfflineReason,
-		ts.PhysLocation,
-		ts.PhysLocationID,
-		ts.Profile,
-		ts.ProfileDesc,
-		ts.ProfileID,
-		ts.Rack,
-		ts.RevalPending,
-		ts.RouterHostName,
-		ts.RouterPortName,
-		ts.Status,
-		ts.StatusID,
-		ts.TCPPort,
-		ts.Type,
-		ts.TypeID,
-		ts.UpdPending,
-		ts.XMPPID,
-		ts.XMPPPasswd,
-	)
-
-	mock.ExpectBegin()
-	mock.ExpectQuery("SELECT").WillReturnRows(rows2)
-	mid, userErr, sysErr, errCode := getMidServers(servers, db.MustBegin())
-
-	if userErr != nil || sysErr != nil {
-		t.Fatalf("getMidServers expected: no errors, actual: %v %v with status: %s", userErr, sysErr, http.StatusText(errCode))
-	}
-	if len(mid) != 1 {
-		t.Fatalf("getMidServers expected: len(mid) == 1, actual: %v", len(mid))
-	}
-	if mid[0].Type != "MID" || *(mid[0].CachegroupID) != 2 || *(mid[0].Cachegroup) != "parentCacheGroup" {
-		t.Fatalf("getMidServers expected: Type == MID, actual: %v", mid[0].Type)
-		t.Fatalf("getMidServers expected: CachegroupID == 2, actual: %v", *(mid[0].CachegroupID))
-		t.Fatalf("getMidServers expected: Cachegroup == parentCacheGroup, actual: %v", *(mid[0].Cachegroup))
-	}
-}
-
 type SortableServers []tc.Server
 
 func (s SortableServers) Len() int {
```

Comment:
I don't think you meant to change this file

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.